### PR TITLE
LPS-138010 Add undoUpdateCollectionDisplayCollection

### PR DESCRIPTION
**Pull request by p2kmgcl**

### CODE REVIEW

File: modules/apps/fragment/fragment-renderer-collection-filter-impl/src/main/java/com/liferay/fragment/renderer/collection/filter/internal/CollectionFilterFragmentRenderer.java [hunk 1]
```diff
@@ -106,10 +107,17 @@ private String _getInfoFilterKey(
...
+			(String)
+				_fragmentEntryConfigurationParser.getConfigurationFieldValue(
+					fragmentEntryLink.getEditableValues(), "filterKey",
+					FragmentConfigurationFieldDataType.STRING);
+
+		if (infoFilterKey == null) {
+			return StringPool.BLANK;
+		}
+
+		return infoFilterKey;
```
Comment (victorg1991) [inline modules/apps/fragment/fragment-renderer-collection-filter-impl/src/main/java/com/liferay/fragment/renderer/collection/filter/internal/CollectionFilterFragmentRenderer.java:120]: you can just simply do 

```java
GetterUtil.getString(_fragmentEntryConfigurationParser.getConfigurationFieldValue(
					fragmentEntryLink.getEditableValues(), "filterKey",
					FragmentConfigurationFieldDataType.STRING)
```

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page-structure/components/item-configuration-panels/CollectionGeneralPanel.js [hunk 2]
```diff
@@ -171,6 +176,45 @@ export const CollectionGeneralPanel = ({item}) => {
 			numberOfItemsPerPage: event.target.value,
 		});
 
+	const shouldPreventCollectionSelect = () => {
+		const state = getState();
+
+		const isLinkedToFilter = Object.values(state.layoutData.items).some(
+			(layoutDataItem) => {
+				if (layoutDataItem.type !== 'fragment') {
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page-structure/components/item-configuration-panels/CollectionGeneralPanel.js:195]: `LAYOUT_DATA_ITEM_TYPES.fragment`?

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/thunks/deleteItem.js [hunk 3]
```diff
@@ -19,9 +19,13 @@ import InfoItemService from '../services/InfoItemService';
 import LayoutService from '../services/LayoutService';
 import getFragmentEntryLinkIdsFromItemId from '../utils/getFragmentEntryLinkIdsFromItemId';
 
-export default function deleteItem({itemId, selectItem = () => {}, store}) {
-	return (dispatch) => {
-		const {fragmentEntryLinks, layoutData, segmentsExperienceId} = store;
+export default function deleteItem({itemId, selectItem = () => {}}) {
+	return (dispatch, getState) => {
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/thunks/deleteItem.js:23]: 😻

**Comment by ealonso** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by p2kmgcl** [comment]
ci:forward

**Comment by p2kmgcl** [comment]
ci:stop

**Comment by p2kmgcl** [comment]
https://github.com/liferay-echo/liferay-portal/pull/5699

**Pull request closed without merging**